updates/next.json

```diff
@@ -87,6 +87,9 @@
     {
       "version": "37.20230303.1.1",
       "metadata": {
+        "barrier": {
```

Comment:
Any doc on what exactly is a barrier release? :eyes:

Reply:
:) - I couldn't find some easily in https://coreos.github.io/zincati/, but basically a barrier release is a release in which all updating nodes before that release will have to update through it.

Take a look at [the update graph](https://builds.coreos.fedoraproject.org/graph?stream=next&basearch=x86_64) and see if you can spot the barrier releases.